config/routes/claims.rb

```diff
@@ -1,17 +1,29 @@
 scope module: :claims, as: :claims, constraints: { host: ENV["CLAIMS_HOST"] } do
-  root to: "pages#index"
+  root to: redirect("/schools")
```

Comment:
Is there a reason why we don't use rails routes helper here?

Reply:
I don't think route helpers are available within the route files, as the route files are used to generate the route helpers.

Reply:
Duh sorry.